.github/workflows/flutter-build.yml

```
@@ -234,7 +234,7 @@ jobs:
           path: rustdesk
 
       - name: Sign rustdesk files
-        if: env.UPLOAD_ARTIFACT == 'true' && env.SIGN_BASE_URL != ''
+        if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)